opendatahub-io/data-science-pipelines-operator
# Remove ViewerCRD deployment

## Description
Remove viewerCRD from Operator code. This is part of the work to make DSPA simpler, deploying only the components that will be part of the Data Science Pipelines release.

## How Has This Been Tested?
* `make run`
* Deploy Any DSPA CR
* Check if the messages to deploy viewerCRD are gone

## Merge criteria:
